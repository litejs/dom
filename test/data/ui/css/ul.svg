<svg xmlns="http://www.w3.org/2000/svg" height="600" width="900" viewBox="-300 -200 900 600" fill="#00a651">
	<path d="M14 340a2.2 2.2 0 1 1 0 .1zm9 20a5.2 5.2 0 1 1 0 .1z"/>
</svg>

