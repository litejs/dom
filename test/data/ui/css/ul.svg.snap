<svg xmlns="http://www.w3.org/2000/svg" height="600" width="900" viewBox="-300 -200 900 600" fill="#00a651">
	<path d="M14 340a2.2 2.2 0 1 1 0 .1zm9 20a5.2 5.2 0 1 1 0 .1z"/>
	<g id="circle-1">
		<circle cx="4" cy="4" r="1"/>
		<circle cx="5" cy="5" r="1"/>
	</g>
	<circle id="circle-2" cx="5" cy="5" r="2"/>
</svg>

